src/reader.rs

```diff
@@ -432,6 +432,21 @@ impl<T: Read + Seek> ShapeReader<T> {
             Err(Error::MissingIndexFile)
         }
     }
+
+    /// Returns the number of the shapes in the shapefile
```

Comment:
There is one too many 'the'.

It should be `Returns the number of shapes in the shapefile`